airbyte-integrations/connectors/source-amazon-ads/integration_tests/spec.json

```diff
@@ -69,6 +69,17 @@
         "items": {
           "type": "integer"
         }
+      },
+      "state_filter": {
+        "title": "State Filter",
+        "description": "Reflects the state of the Display Campaign stream as enabled, paused, or archived. If you do not populate this field, it will be ignored completely",
```

Comment:
The acceptance test doesn't pass as spec.yaml and spec.json still don't match. So please update it carefully and run acceptance tests locally. See my [previous comment](https://github.com/airbytehq/airbyte/pull/15985/files#r960285013) on how to do that.

Reply:
Do not edit this file manually. Use YAML to JSON converter and run `./gradlew format` to match code style. Manual editing is prone to repeated mistakes.

Reply:
sounds good - used a YAML to JSON converter and ran ./gradlew format

Reply:
Great, the test is finally passing.